registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -16,10 +16,10 @@
       "iconUrls": {
         "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/rmrk.svg"
       },
-      "deployer": "0x0000000000000000000000000000000000000000",
-      "originalMinter": "0x0000000000000000000000000000000000000000",
-      "deploySalt": "0x",
-      "deploymentMessageId": "0x0000000000000000000000000000000000000000000000000000000000000000-0",
+      "deployer": "0xCD7A0D098E3A750126b0fec54BE401476812cfc0",
+      "originalMinter": "",
```

Comment:
do we prefer to allow empty fields or not having the key at all?

Reply:
i think a null field is good, shows it's intentionally empty (vs. having a missing property). in the schema, it's null or legit address, so i'll make the slight update